src/splitInteger.test.js

```diff
@@ -4,18 +4,16 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
-});
-
-test(`should return a part equals to a value
-  when splitting into 1 part`, () => {
-
+  expect(splitInteger(17, 4))
+    .toHaveLength(4);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(32, 6))
+    .toEqual([5, 5, 5, 5, 6, 6]);
 });
 
 test('should add zeros if value < numberOfParts', () => {
-
+  expect(splitInteger(17, 0))
+    .toEqual([]);
```

Comment:
The test case for `splitInteger(17, 0)` is incorrect. According to the task description, the function should return an array with exactly `numberOfParts` elements. Since `numberOfParts` is 0, this test case is invalid as per the task requirements.